src/context.tsx

```diff
@@ -43,4 +39,49 @@ const init = function(){
     if (storage) state.fallenHeroes= JSON.parse(storage)
 }
 init()
-console.log(state.fallenHeroes, state.restOfAliveHeroes)
\ No newline at end of file
+console.log(state.fallenHeroes, state.restOfAliveHeroes)
+
+*/
+
+export interface CampaignContextInterface {
+  campaign: Campaign;
+  setCampaign: Dispatch<SetStateAction<Campaign>>;
+}
+
+const defaultState = {
+  campaign: {
+    fallenHeroes: [],
+    restOfAliveHeroes: ["Aragorn", "Eowyn", "Beravor"],
+    allHeroes: ["Aragorn", "Eowyn", "Beravor"],
+  },
+  setCampaign: (campaign: Campaign) => {},
+} as CampaignContextInterface;
+
+const LotrContext = React.createContext(defaultState);
+
+type CampaignLotrProvider = {
+  children: ReactNode;
+};
+
+function LotrProvider({ children }: CampaignLotrProvider) {
+  const [campaign, setCampaign] = useState<Campaign>({
+    fallenHeroes: [],
+    restOfAliveHeroes: ["Aragorn", "Eowyn", "Beravor"],
+    allHeroes: ["Aragorn", "Eowyn", "Beravor"],
+  });
+
+  // error
+  const [error, setError] = useState({ show: false, msg: "" });
```

Comment:
Unused.

Reply:
I think we will use something like that in the future, but ok, removing for now..